modules/core/src/test/java/org/apache/ignite/internal/processors/cache/GridCacheFullTextQueryMultithreadedSelfTest.java

```diff
@@ -76,6 +76,7 @@ public void testH2Text() throws Exception {
         final int keyCnt = 5000;
         final int logFreq = 50;
         final String txt = "Value";
+        final int limit = 1000;
```

Comment:
Not sure that we should change the behavior in old test. What are the arguments to do it?

Reply:
Yes, there's a reason not to set limits here, as we have created special tests for this.